Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -0,0 +1,545 @@
+        cy.get("[data-testid=btnCancel]").should("be.visible");
+        cy.get("[data-testid=btnEnter]").should("be.enabled", "be.visible");
+        cy.get("[data-testid=btnPrivacyStatement]").should("be.visible");
+        cy.get("[data-testid=btnLogin]").should("be.enabled", "be.visible");
+    });
+
+    it("Vaccination Card - Cancel - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        cy.get("[data-testid=btnCancel]").should("be.visible").click();
+
+        cy.get("[data-testid=btnVaccineCard]").should("exist");
+    });
+
+    it("Vaccination Card - Login BC Services Card App - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        cy.get("[data-testid=btnLogin]")
+            .should("be.visible", "be.enabled")
+            .click();
+
+        cy.get("[data-testid=BCSCBtn]").should("be.visible", "be.enabled");
+        cy.get("[data-testid=IDIRBtn]").should("be.visible", "be.enabled");
+        cy.get("[data-testid=KeyCloakBtn]").should("be.visible", "be.enabled");
+    });
+
+    it("Vaccination Card - Valid PHN via Select Year - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dobYearSelector, "Year");
+
+        cy.get("[data-testid=phnInput]")
+            .should("be.visible", "be.enabled")
+            .and("have.class", "form-control is-valid");
+    });
+
+    it("Vaccination Card - Invalid PHN via Select Year - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn").replace(/.$/, ""));
+
+        select(dobYearSelector, "Year");
+
+        cy.get(feedbackPhnMustBeValidSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - PHN, DOB and DOV not entered via Click Enter - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackPhnIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOB and DOV not entered with Invalid PHN via Click Enter - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn").replace(/.$/, ""));
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackPhnMustBeValidSelector).should("be.visible");
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
```

Comment:
I don't know if we need to check these two fields here since they've already been checked in the previous test.

Reply:
I want to make sure all three messages get displayed when DOB, DOV and PHN validation errors occur at the same time.

Reply:
As discussed, it may be easier to maintain if the individual validation tests only check one field at a time. It is fine either way, though.